Merkll/Quick-Credit
# 166007388 Build modules for PostgreSQL integration with data Model

#### What does this PR do?
Build modules for PostgreSQL integration with data Model

#### Description of Task to be completed?
Data persistence using PostgreSQL

#### How should this be manually tested?
Clone Repo
Run ```npm install```
Ensure database is up and running and specify credentials as an environmental variable
RUN ```npm start```
Test All endpoint and see data persisted in the database

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Relevant Pivotal Tracker Stories
166007388
